src/include/compat.h

```diff
@@ -79,8 +79,16 @@
 /* get PATH_MAX */
 #include <limits.h>
 
+#ifndef EUCLEAN
+#define EUCLEAN         117	/* Structure needs cleaning */
```

Comment:
could you drop the extra spaces? and drop the comments also. as it's not our responsibility to document the errno. whoever interested in it should consult the manpage, for instance, http://man7.org/linux/man-pages/man3/errno.3.html .

Reply:
@tchaikov 
Will do, I'll just ad the URl as comment.

Reply:
@wjwithagen i'd suggest not.

Reply:
@tchaikov 
Perhaps it is a note to myself, as I'm regularly looking for the place where things like these are defined. But I'll remove